tests/helpers/test_script.py

```diff
@@ -1672,74 +1071,70 @@ def wait_started_cb():
         await hass.async_block_till_done()
 
         assert not script_obj.is_running
-        assert len(events) == 3
-        assert events[2].data["value"] == 2
+        assert len(events) == 2 + len(last_events)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5164426)